documents/operating.md

```diff
@@ -21,10 +21,100 @@ SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
+6. Help maintain and further the BIFFUD public footprint.
+7. Try to attend BIFFUD events.
+
+### Elected Roles
+In addition to base expectations, the following positions are to be held
+by corporate overlords.  They are elected annually:
+
+- **President:** Has a gavel.  The President is responsible for merging
+approved pull requests to `\corporate`.
+
+- **Vice President:** Has a smaller gavel.  The Vice President is responsible
+for laughing whenever the President makes a joke.
+
+- **Treasurer:** Keeps track of money, works with accountants, oversees
+and approves small expenses.
+
+- **Secretary:** Ensures notes and minutes are taken for meetings,
+ensures that proper CONTRIBUTING.md protocols are followed in `/corporate`.
+
+- **Beastmaster:** Ensures that the corporate overlords are aware of the
+progress and activity of BIFFUD projects.
+
+These roles are responsible for overseeing their duties, though in some
+cases that responsibility can take the form of setting up processes and
+support networks.  For instance, the Beastmaster can delegate oversight
+over a subset of BIFFUD projects if we have too many.
+
+### Adding Corporate Overlords
+Although previously thought impossible, an ancient ritual found buried
+in the ruins of the Lawyer Cave describes the process of creating new
+BIFFUD Corporate Overlords.
+
+Any Corporate Overlord may nominate the addition of a new Overlord during
+a plotting session.  If the nomination is seconded, the ritual begins.
+
+1. The Corporate Overlords are presented with a biography of the candidate.
+2. The candidate submits a piece of original poetry or other creative work
+for the Overlords' enjoyment and consideration.
+3. A vote is held.
+
+If 70% or more of the Corporate Overlords approve the addition, then after the
+plotting session ends the new Corporate Overlord is onboarded and ownership of
+the company is redistributed so that each overlord owns an equal share.
+
+
+### Removing Corporate Overlords
+A Corporate Overlord can recuse itself from BIFFUD at any time by dying, uploading
+its consciousness to the cloud, or submitting a letter of resignation to both
+the President of BIFFUD and the President of the United States.
+
+Corporate Overlords can also vote to remove a peer from the organization if
+any of the following conditions apply:
+
+- The Corporate Overlord fails to attend 50% of the monthly plotting sessions
+in a given year without a note from a family member explaining its absence.
+- The Corporate Overlord has violated a serious federal law in the past year.
+- The Corporate Overlord is hated or despised.
+
+The removal of a corporate overlord can be nominated during a plotting session
+and requires 70% approval or more from the other Corporate Overlords.
+
+Overlords are entitled to a pro-rated profit share for their tenure in a given
+year, as well as ongoing profit-share according to their ownership of various
+BIFFUD projects.
```

Comment:
👍  -- as far as the year question we should be explicit that by "year" we mean fiscal year -- profits / taxes / etc are calculated at the end of each fiscal year.

One concern around pro rating based on when the profits come in is that (because aren't paying accountants to do monthly reports) we are doing single reports each year.  This means our granularity / ability to calculate exactly what was earned before a person left will be harder to calculate, especially since we still need to figure out what % is removed for taxes for partners, expenses incurred for a given project over the full year, etc.  (if a project had more expenses in the first half than the second half, but profits were on the second half not the first half.... etc.)

I hear the equity argument, I just think it will be too hard to execute on precisely.  Dividing the profits you would have been entitled to by the % of the fiscal year you were a partner is pretty simple.

Also, either way we should explicitly note that the remaining % would be given back to the company.